src/open_inwoner/templates/pages/profile/notifications.html

```diff
@@ -9,6 +9,23 @@ <h1 class="h1" id="title">
     {% trans "Kies voor welk onderwerp je meldingen wilt ontvangen" %}
 </p>
 
-{% form form_object=form method="POST" id="change-notifications" submit_text=_("Opslaan") secondary_href='profile:detail' secondary_text=_('Terug') secondary_icon='arrow_backward' secondary_icon_position="before" %}
+<form method="POST" id="change-notifications" action="{% url 'profile:notifications' %}" class="form" novalidate>
+    {% csrf_token %}
+
+    {% for field in form.fields %}
+        {% autorender_field form field %}
+    {% endfor %}
+
+    <div class="form__control">
```

Comment:
Maybe we could move this disabled checkbox to a separate template that can be included, to avoid duplication.

Also, is there a reason why this is a `<span role="checkbox">` and not an `<input type="checkbox" disabled>`?

Reply:
Let's not make a reusable template for bad patterns such as actual disabled inputs that cannot be enabled by the user, as I explained in the description of this PR above - also: [new designs are coming soon](https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=1373%3A5943&mode=design&t=na0I2hZEz4Y4U2c0-1) and this PR is an intermediate change.
Also: yes, this should _not_ be an actual input tag, because accessibility devices completely ignore disabled inputs and do not give any textual/label information.

Reply:
check, didn't know that :+1: